services/s3control/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
We shouldn't add `log4j.properties` in the main resources

Reply:
I thought I added it in the test directory, will move it.